tools/dds/dds-adapter/lrs-device-controller.cpp

```diff
@@ -198,26 +216,19 @@ std::vector< std::shared_ptr< realdds::dds_stream_server > > lrs_device_controll
             continue;
         }
 
-        // Set stream metadata support (currently if the device supports metadata all streams does)
-        // Must be done before calling init_profiles()
-        if( _md_enabled )
-        {
-            server->enable_metadata();
-        }
-
-        server->init_profiles( profiles, default_profile_index );
-
         // Set stream intrinsics
-        auto video_server = std::dynamic_pointer_cast< dds_video_stream_server >( server );
-        auto motion_server = std::dynamic_pointer_cast< dds_motion_stream_server >( server );
-        if( video_server )
+        if( auto video_server = std::dynamic_pointer_cast<dds_video_stream_server>(server) )
         {
             video_server->set_intrinsics( std::move( stream_name_to_video_intrinsics[stream_name] ) );
+            // Set stream metadata support (currently if the device supports metadata all streams does)
```

Comment:
Comment correct to video streams only

Reply:
Yeah but it's inside a block only for video streams... You want me to change?